models/intel/yolo-v2-ava-0001/description/yolo-v2-ava-0001.md

```diff
@@ -9,25 +9,26 @@ This is a reimplemented and retrained version of the [YOLO v2](https://arxiv.org
 | Metric                          | Value                                     |
 |---------------------------------|-------------------------------------------|
 | Mean Average Precision (mAP)    | 63.9%                                     |
-| Flops                           | 48.29Bn\*                                  |
+| Flops                           | 48.29Bn\*                                 |
```

Comment:
I think we need align flops formatting with other models

Reply:
Updated